.github/workflows/build.yml

```diff
@@ -96,16 +96,22 @@ jobs:
           path:
             build/reports
 
+      - name: Get Jacoco report paths
+        id: jacoco-paths
+        run: |
+          paths=$(find ${{ github.workspace }} -name jacocoTestReport.xml | tr '\n' ',')
+          echo "paths=$paths" >> "$GITHUB_OUTPUT"
       - name: Jacoco Report to PR
         id: jacoco
-        uses: madrapps/jacoco-report@v1.6.1
+        uses: yijhenlin/jacoco-report@summary_mode
```

Comment:
So you build your own github action?

Reply:
hi @yijhenlin Thank you for your contribution.
You fixed jacoco issue in your github repo? https://github.com/yijhenlin/jacoco-report/commit/4305a5aece22410ce20701949be29c9c887cb111

Reply:
Yes, I fork from madrapps/jacoco-report and patch an option to check if this idea works or not, since it didn't expose the report's raw data to user level.
I'll try to contribute back to madrapps/jacoco-report if the workaround is accepted and you can fork your own too.